hack/build-images.sh

```diff
@@ -138,6 +138,9 @@ image openshift/origin-sti-builder           images/builder/docker/sti-builder
 image openshift/origin-f5-router             images/router/f5
 image openshift/node                         images/node
 
+# images used for tests
+image openshift/test-multicast               images/testing/multicast
```

Comment:
I'm not really sure. Maybe @smarterclayton has an opinion here. Maybe mounting in binaries to an image we pull down would be better?